Css/WebScrap.Css/Listeners/CssTagsListener.cs

```diff
@@ -38,16 +40,15 @@ internal override void Process(ClosingTag tag)
 
     private bool IsNameMet(TagBase tag)
     {
-        var css = cssTracker.GetCurrentExpectedTag(cssTags);
-        return tag.Name.AsSpan()
-            .SequenceEqual(css.Tag.Span);
+        var css = cssTracker.GetCurrentExpectedTag(cssTags.Count);
+        return tag.Name.Equals(css.Name);
     }
 
     private bool IsAttrMet(OpeningTag tag)
     {
-        var css = cssTracker.GetCurrentExpectedTag(cssTags);
+        var css = cssTracker.GetCurrentExpectedTag(cssTags.Count);
```

Comment:
It doesn't really makes sense. Why cssTags.Count? I cant tell by looking at the call.